### ISSUE TITLE
Effective NPE in `AbstractLSPFeature.getClientFeatures()`

### ISSUE THREAD

**Issue by SCWells72** [redhat-developer/lsp4ij#657]
This is admittely in 0.7.0 and may be fixed in 0.8.0, but I'm seeing this unhandled exception with some frequency:

```
java.lang.IllegalStateException: @NotNull method com/redhat/devtools/lsp4ij/client/features/AbstractLSPFeature.getClientFeatures must not return null
	at com.redhat.devtools.lsp4ij.client.features.AbstractLSPFeature.$$$reportNull$$$0(AbstractLSPFeature.java)
	at com.redhat.devtools.lsp4ij.client.features.AbstractLSPFeature.getClientFeatures(AbstractLSPFeature.java:37)
	at com.redhat.devtools.lsp4ij.client.features.LSPSemanticTokensFeature.getTextAttributesKey(LSPSemanticTokensFeature.java:41)
	at com.redhat.devtools.lsp4ij.features.semanticTokens.SemanticTokensData.highlight(SemanticTokensData.java:110)
	at com.redhat.devtools.lsp4ij.features.semanticTokens.LSPSemanticTokensHighlightVisitor.highlightSemanticTokens(LSPSemanticTokensHighlightVisitor.java:97)
	at com.redhat.devtools.lsp4ij.features.semanticTokens.LSPSemanticTokensHighlightVisitor.analyze(LSPSemanticTokensHighlightVisitor.java:59)
```

I'm not sure if it should be allowed for `getClientFeatures()` to return `null` and the return type annotation should be changed to `@Nullable` (and all callers updated to allow for a `null` return value), or if there's an underlying issue that's leading to it trying to return a `null` value for `clientFeatures`.

**Comment by angelozerr** [comment]
It should be fixed in 0.8.0

**Comment by SCWells72** [comment]
Is there another bug that this one is a duplicate of? Or a commit/PR that fixed it that might be linked here?

**Comment by angelozerr** [comment]
It was fixed in 0.8.1
